composer.json

```diff
@@ -20,14 +20,18 @@
   },
   "require": {
     "ext-dom": "*",
+    "ext-gmp": "*",
```

Comment:
Could we remove this then if it's not our dependency. This should then be added to the Stepup-Bundle.